### PR TITLE
feat: add LFM2.5-8B-A1B recipes

### PR DESCRIPTION
**Pull request by ykhrustalev**
**Problem**

The GenAI model builder is gaining LFM2-MoE export in microsoft/onnxruntime-genai#2575, but nothing here builds LFM2.5-8B-A1B. The recipes cannot be run without a build that contains it.

**Solution**

- Adds `cpu/` and `cuda/` recipes at int4 and int8 for `LiquidAI/LFM2.5-8B-A1B`, registered as `arch: lfm2_moe`.
- Pins `qmoe_block_size: 32` for the fused `QMoE` experts; at 128 the same recipe loses 40% on KL and 2.2 points of perplexity to save 0.34 GiB.
- Keeps the k_quant and `matmul_mixed_precision` settings of the LFM2.5 text recipes for int4: against plain int4 they gain 3.3 points of top-1 on CPU (McNemar p = 0.005) and lower KL on both EPs.
- Builds int8 with the model builder alone, without the `rtn` pass the dense LFM2.5 int8 recipes use, because that pass cannot walk an MoE model (`Lfm2MoeForCausalLM` has no `model.layers.N.mlp`).
- Records measured size, quality and decode speed per recipe in each README.

**Testing**

Exported every recipe on an A10 with a 30-core Xeon 8358, against a build of onnxruntime-genai#2575 and an ONNX Runtime containing microsoft/onnxruntime#32644.

- Scored each export against Hugging Face fp32 logits over 819 positions from 8 chat prompts (prose, code, arithmetic, history, documentation, data structures, translation, systems), comparing variants with paired McNemar on top-1 and a paired bootstrap on KL, since single-prompt differences are inside the noise.
- Re-exported each recipe from the checked-in JSON and compared sha256 against the artifact that was measured, so the numbers below describe these files.
- Verified the graphs structurally: 22 `QMoE` ops on layers 2-23, block size 32, the router left as a float `MatMul`, and the int8 promotions landing where `matmul_mixed_precision` selects them.

| recipe | size | top-1 | KL | decode |
| --- | --- | --- | --- | --- |
| cuda int8 | 8.9 GiB | 0.955 | 0.0175 | 164 t/s |
| cuda int4 | 5.1 GiB | 0.880 | 0.0983 | 202 t/s |
| cpu int8 | 9.9 GiB | 0.96 | 0.009 | 45 t/s |
| cpu int4 | 6.1 GiB | 0.891 | 0.0852 | 51 t/s |

CUDA int8 is statistically indistinguishable from fp16 (0.959 top-1, KL 0.0224, 15.8 GiB) at 56% of its size: McNemar p = 0.65 and the paired KL interval spans zero. `moe_quant_type=int8` was also measured and is dominated by plain int8, so it is not a recipe here.

### CODE REVIEW

File: LiquidAI-LFM2.5-8B-A1B/cpu/README.md [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+are quantized block-wise by the builder itself through `qmoe_block_size`. The router MatMul
+is always left in floating point so quantization rounding cannot change which experts a
+token is sent to.
+
+## Recipes
+
+### `_cpu_int8.json` — closest to the original
+Symmetric INT8 weights and INT8 experts, straight from the model builder. Use this when
+10 GiB is acceptable: it costs about 60% more disk than INT4 and roughly 10% of the decode
+speed, and it is far closer to the original model.
```
[the remark this answers is not in the document]
Comment (ykhrustalev) [reply LiquidAI-LFM2.5-8B-A1B/cpu/README.md]: Good catch — that was a wording error, not a measurement one. INT8 decodes at 45 tok/s against INT4's 51, so it is about 10% slower. Reworded to "decodes about 10% slower".
